cypress/e2e/bulk-edit/in-app/verify-suppress-from-discovery-option-in-case-holdings-not-suppressed-items-suppressed-set-false.cy.js

```diff
@@ -0,0 +1,151 @@
+import permissions from '../../../support/dictionary/permissions';
+import TopMenu from '../../../support/fragments/topMenu';
+import Users from '../../../support/fragments/users/users';
+import BulkEditSearchPane from '../../../support/fragments/bulk-edit/bulk-edit-search-pane';
+import BulkEditActions from '../../../support/fragments/bulk-edit/bulk-edit-actions';
+import getRandomPostfix from '../../../support/utils/stringTools';
+import BulkEditFiles from '../../../support/fragments/bulk-edit/bulk-edit-files';
+import TopMenuNavigation from '../../../support/fragments/topMenuNavigation';
+import InventoryInstances from '../../../support/fragments/inventory/inventoryInstances';
+import FileManager from '../../../support/utils/fileManager';
+import HoldingsRecordView from '../../../support/fragments/inventory/holdingsRecordView';
+import InventorySearchAndFilter from '../../../support/fragments/inventory/inventorySearchAndFilter';
+import ItemRecordView from '../../../support/fragments/inventory/item/itemRecordView';
+import InventoryItems from '../../../support/fragments/inventory/item/inventoryItems';
+
+let user;
+const instanceHRIDFileName = `instanceHRID_${getRandomPostfix()}.csv`;
+const item = {
+  instanceName: `testBulkEdit_${getRandomPostfix()}`,
+  itemBarcode: getRandomPostfix(),
+};
+const matchedRecordsFileName = `*-Matched-Records-${instanceHRIDFileName}`;
+const previewOfProposedChangesFileName = `*-Updates-Preview-${instanceHRIDFileName}`;
+
+describe('bulk-edit', () => {
+  describe('in-app approach', () => {
+    before('create test data', () => {
+      cy.createTempUser([
+        permissions.inventoryAll.gui,
+        permissions.bulkEditView.gui,
+        permissions.bulkEditEdit.gui,
+        permissions.bulkEditLogsView.gui,
+      ]).then((userProperties) => {
+        user = userProperties;
+
+        item.instanceId = InventoryInstances.createInstanceViaApi(
+          item.instanceName,
+          item.itemBarcode,
+        );
+        cy.getItems({ query: `"barcode"=="${item.itemBarcode}"` }).then((inventoryItem) => {
+          inventoryItem.discoverySuppress = true;
+          item.itemId = inventoryItem.id;
+          InventoryItems.editItemViaApi(inventoryItem);
+        });
+        cy.getHoldings({
+          limit: 1,
+          expandAll: true,
+          query: `"instanceId"="${item.instanceId}"`,
+        }).then((holdings) => {
+          item.holdingsHRID = holdings[0].hrid;
+          cy.updateHoldingRecord(holdings[0].id, {
+            ...holdings[0],
+          });
+        });
+        cy.getInstanceById(item.instanceId).then((body) => {
+          cy.updateInstance(body);
```

Comment:
what exactly is the update here? getting holdings and instances is unnecessary, since you only need to get and put items

Reply:
Removed